lte/gateway/python/magma/pipelined/app/BUILD.bazel

```diff
@@ -204,6 +204,9 @@ py_library(
 py_library(
     name = "restart_mixin",
     srcs = ["restart_mixin.py"],
+    deps = [
```

Comment:
NIT for one liner lists we usually do not indent, i.e., `deps = [":startup_flows"],`